composer.json

```diff
@@ -19,39 +19,41 @@
         "erusev/parsedown": "^1.6",
         "sensio/framework-extra-bundle": "^6.1",
         "symfony/apache-pack": "^1.0",
-        "symfony/asset": "^5.3",
-        "symfony/console": "^5.3",
-        "symfony/dotenv": "^5.3",
-        "symfony/expression-language": "^5.3",
+        "symfony/asset": "5.4.0-beta1",
+        "symfony/console": "5.4.0-beta1",
+        "symfony/dotenv": "5.4.0-beta1",
+        "symfony/expression-language": "5.4.0-beta1",
         "symfony/flex": "^1.1",
-        "symfony/form": "^5.3",
-        "symfony/framework-bundle": "^5.3",
-        "symfony/intl": "^5.3",
-        "symfony/mailer": "^5.3",
+        "symfony/form": "5.4.0-beta1",
+        "symfony/framework-bundle": "5.4.0-beta1",
+        "symfony/intl": "5.4.0-beta1",
+        "symfony/mailer": "5.4.0-beta1",
         "symfony/monolog-bundle": "^3.1",
         "symfony/polyfill-intl-messageformatter": "^1.12",
-        "symfony/security-bundle": "^5.3",
-        "symfony/runtime": "^5.3",
-        "symfony/string": "^5.3",
-        "symfony/translation": "^5.3",
-        "symfony/twig-pack": "^1.0",
-        "symfony/validator": "^5.3",
+        "symfony/runtime": "5.4.0-beta1",
+        "symfony/security-bundle": "5.4.0-beta1",
+        "symfony/string": "5.4.0-beta1",
+        "symfony/translation": "5.4.0-beta1",
+        "symfony/twig-bundle": "5.4.0-beta1",
+        "symfony/validator": "5.4.0-beta1",
         "symfony/webpack-encore-bundle": "^1.4",
-        "symfony/yaml": "^5.3",
+        "symfony/yaml": "5.4.0-beta1",
         "tgalopin/html-sanitizer-bundle": "^1.2",
+        "twig/extra-bundle": "^2.12|^3.0",
         "twig/intl-extra": "^3.0",
-        "twig/markdown-extra": "^3.0"
+        "twig/markdown-extra": "^3.0",
+        "twig/twig": "^2.12|^3.0"
```

Comment:
allowing 2.12 here does not make sense when you extra packages are requiring the 3.0 versions which depend on Twig 3.

Reply:
Correct! This leftover has been removed.

Reply:
@javiereguiluz this does not seem removed